### DISCUSSION TITLE
Añadir prerequisitos para el Dashboards.sh

### DISCUSSION THREAD

**Discussion by hectorrf16** [CarlosCuezva/dashboards-Grafana-Teslamate#7] [Ideas]
Hola Carlos, primero de todo, gracias por los Dashboards... te sigo en twitter y he estado añadiendo los dashboards manualmente, pero al final me canse de hacerlo y decidí utilizar tu .sh

He visto que en mi nueva maquina el JQ no estaba como paquete instalado, por lo que el script fallaba a la hora de escribir, que tal si añadimos un "dpkg-query -l jq | grep 'jq'" o algo similar (idea muy básica y rápida que se me ha ocurrido ahora) para verificar que el paquete este instalado y así que tire el código?

**Comment by CarlosCuezva** [comment]
Hola Hector.

Normal que te hayas cansado de hacerlo a mano jajaja.

Pues tienes más razón que un santo así que a ver si me pongo a buscarle una solución "elegante" al tema. 

Gracias!!!!

**Comment by CarlosCuezva** [comment]
Acabo de subir una actualización del script de instalación donde se ha eliminado la dependencia de JQ, asi que todavía mejor

Commit: [Removed dependency of jq program](https://github.com/CarlosCuezva/dashboards-Grafana-Teslamate/commit/a87faaaba203a5cbc11b6657e4a1b9377b3f477c)
